config/config.ts

```diff
@@ -24,75 +24,73 @@ interface EnvironmentConfig {
 /**
  * The function will return the appropriate configuration for the given account
  *
- * @param accountName the name of account stage
- * @returns the configuration for the given accountName
+ * @param s the name of account stage
```

Comment:
I find single letter parameters unnecessary cryptic? Why not `stage`?

Reply:
Yup, updated!